src/main/java/com/cloudogu/scm/mirror/api/RepositoryTypeEnricher.java

```diff
@@ -54,6 +57,11 @@ public void enrich(HalEnricherContext context, HalAppender appender) {
     if (repositoryType.getSupportedCommands().contains(Command.MIRROR)) {
       String createUrl = new LinkBuilder(scmPathInfoStore.get().get(), MirrorRootResource.class).method("createMirrorRepository").parameters().href();
       appender.appendLink("mirror", createUrl);
+      if (repositoryType.getName().equals("git") && !configurationService.getGlobalConfiguration().isDisableRepositoryFilterOverwrite()) {
+        String filterConfigurationLink = new LinkBuilder(scmPathInfoStore.get().get(), MirrorRootResource.class, MirrorResource.class)
+          .method("repository").parameters("{namespace}", "{name}").method("getFilterConfiguration").parameters().href();
+        appender.appendLink("mirrorFilterConfiguration", filterConfigurationLink);
+      }
```

Comment:
We do not have any good options here, but is it a good idea to add a link to decide, whether or not to display the filter part? Wouldn't it be better to put this in the frontend? However we do this, we will have to change to plugin, for example when we add Mercurial. But a link may raise questions and expectations.

Reply:
I spoke to @sdorra briefly about this and he signed off on this solution. I usually prefer the UI to be driven by the backend wherever possible. Which questions and/or expectations do you expect that could be problematic ?

Reply:
It was just that we have another rest api call that we do not use. But I'm fine with this.